cmd/loom/loom.go

```diff
@@ -617,14 +617,7 @@ func loadAppStore(cfg *config.Config, logger *loom.Logger, targetVersion int64)
 		}
 	}
 
-	if cfg.CachingStoreConfig.CachingEnabled &&
-		((cfg.AppStore.Version == 1) || cfg.CachingStoreConfig.DebugForceEnable) {
-		appStore, err = store.NewCachingStore(appStore, cfg.CachingStoreConfig)
-		if err != nil {
-			return nil, err
-		}
-		logger.Info("CachingStore enabled")
-	} else if cfg.CachingStoreConfig.CachingEnabled && cfg.AppStore.Version == 3 {
+	if cfg.CachingStoreConfig.CachingEnabled && cfg.AppStore.Version == 3 {
```

Comment:
AppStore v1 is now compatible with `VersionedCachingStore` too.

Reply:
AppStore v1 Support is added.